src/main/java/com/netgrif/application/engine/importer/service/FieldFactory.java

```diff
@@ -550,6 +558,15 @@ private void parseUserValues(UserField field, Case useCase, String fieldId) {
         field.setValue((UserFieldValue) useCase.getFieldValue(fieldId));
     }
 
+    private void parseUserListValues(Field field, Case useCase, String fieldId) {
+        List<UserFieldValue> userFieldValues = ((List<String>) useCase.getFieldValue(fieldId)).stream().map(id -> {
+            IUser user = userService.findById(id, false);
```

Comment:
duplicate creation of UserFieldValue from user id. The `DataService` implements a method for this.

Reply:
Resolved